spec/features/chefs/show_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe "Chef show page" do
+  before do
+    @chef = Chef.create(name: "Gordon Ramsay")
+  end
+
+  it "displays the chef's name" do
+    visit "/chefs/#{@chef.id}"
+    expect(page).to have_content("Gordon Ramsay")
+  end
+
+  it "has a link to view the chef's ingredients" do
+    visit "/chefs/#{@chef.id}"
+    expect(page).to have_link("View Ingredients")
+  end
```

Comment:
When you have a `have_link` test like this, you always want to make sure to click the link and assert that the redirect goes to where you expect it to go.

```ruby
click_link "View Ingredients"
expect(current_path).to eq("/chefs/#{@chef.id}/ingredients")
```